### PR TITLE
Don't show report an issue for enterprise 500s

### PR DESCRIPTION
**Pull request by orangejenny**
https://trello.com/c/0Uugx3Xz/152-being-able-to-stop-users-who-have-edit-worker-role-from-assigning-an-admin-role

"don't merge" tag because I want to double-check whether they want the whole help menu hidden or just the report an issue

code buddy @nickpell 
icds @snopoke

### CODE REVIEW

**Review by nickpell — APPROVED** [review]

**Review by snopoke — APPROVED** [review]

File: corehq/util/context_processors.py [hunk 1]
```diff
@@ -37,8 +37,14 @@ def get_per_domain_context(project, request=None):
     ):
         custom_logo_url = reverse('logo', args=[project.name])
 
+    report_an_issue = True
+    if (hasattr(request, 'couch_user') and request.couch_user and project
+        and not request.couch_user.has_permission(project.name, 'report_an_issue')
+    ):
```
Comment (dimagimon) [inline corehq/util/context_processors.py]: E124 closing bracket does not match visual indentation

File: corehq/util/context_processors.py [hunk 2]
```diff
@@ -33,12 +33,16 @@ def is_commtrack(project, request):
 def get_per_domain_context(project, request=None):
     custom_logo_url = None
     if (project and project.has_custom_logo
-        and domain_has_privilege(project.name, privileges.CUSTOM_BRANDING)
-    ):
+        and domain_has_privilege(project.name, privileges.CUSTOM_BRANDING)):
```
Comment (dimagimon) [inline corehq/util/context_processors.py]: E129 visually indented line with same indent as next logical line

File: corehq/util/context_processors.py [hunk 3]
```diff
@@ -33,12 +33,16 @@ def is_commtrack(project, request):
...
         custom_logo_url = reverse('logo', args=[project.name])
 
+    report_an_issue = True
+    if (hasattr(request, 'couch_user') and request.couch_user and project
+        and not request.couch_user.has_permission(project.name, 'report_an_issue')):
```
Comment (dimagimon) [inline corehq/util/context_processors.py]: E129 visually indented line with same indent as next logical line

**Comment by orangejenny** [comment]
Updated to include another part of the card: an `report_an_issue` permission that defaults to true but can be removed from roles. (doing it in another PR would have meant merge conflicts)

**Comment by orangejenny** [comment]
Pinging for review; I added a bit of code since y'all last looked at this. If those changes look good, this is ready to merge.

**Pull request merged**